src/gateways/googleAnalyticsClient.ts

```diff
@@ -102,6 +102,7 @@ export const campaignFromQuery: (query: Query) => SetPayload = flow(
 class GoogleAnalyticsClient {
   private trackerNames: string[] = [];
   private pendingCommands: PendingCommand[] = [];
+  private trackingIsDisabled: boolean = !!Cookies.get(disableAnalyticsCookie);
```

Comment:
This checks for the cookie only on the initialization of the class.
The cookie in theory may be set also in the middle of work so I think it should be a method